components/x-audio/src/components/styles.scss

```diff
@@ -57,8 +57,8 @@ $image-size: 72px;
 }
 
 .audio-player--minimised {
-	grid-template-columns: $height--minimised min-content auto $height--minimised;
-	grid-template-areas: "play-pause series-name title close-button"
+	grid-template-columns: $height--minimised auto auto $height--minimised;
+	grid-template-areas: "play-pause series-name series-name close-button"
 	"play-pause status status close-button";
```

Comment:
This grid can be less `columns` like below because we don't need to separate series-name and title anymore.
```
  grid-template-columns: $height--minimised auto $height--minimised;
  grid-template-areas: "play-pause series-name close-button"
  "play-pause status close-button";
```

Reply:
Very good point, thanks!